### ISSUE TITLE
train refinenet problem

### ISSUE THREAD

**Issue by liuyiqunfaker** [rajat95/Deep-Deghosting-HDR#8]
i trained the refinenet used the kalantari_dataset (input_flies, i did not use the aligned files). i wonder if you used the train_refine.py to train the model,  i concat the optical flow as the input , which you used in infer.py , i have trained(55001,155001) iters ,but i don't think the result is good as the pretrained model , here is my output for ManStanding 
![map1](https://user-images.githubusercontent.com/50859697/97826848-aece6e80-1cfd-11eb-8136-e8d968ddec75.jpg)
![map2](https://user-images.githubusercontent.com/50859697/97826863-b7bf4000-1cfd-11eb-8b48-e4ac2aa4b30d.jpg)

**Comment by rajat95** [comment]
It is hard to say what may be the issue here. Can you share your loss curves and maybe I can help you out?

**Comment by liuyiqunfaker** [comment]
![2020-11-03 13-43-33屏幕截图](https://user-images.githubusercontent.com/50859697/97954799-7606c600-1ddf-11eb-8133-243f9fac5670.png)
here is the train loss curve

**Comment by rajat95** [comment]
If  I am understanding you correctly you are finetuning our pretrained model on Kalantari et al's dataset right? We didnt have access to ground truth static images and collected our own dataset and used the scripts provided to train the refinement net. For different dataset you may need to experiment with hyper-parameter values. Also, its possible that our dataset is better suited to train the refinement model and you may not get as good result as we get from our dataset. In any case, looking at the loss curve I would suggest starting with a slightly lower learning rate and decreasing it in a step by step fashion only after  the loss curve stagnates

**Comment by liuyiqunfaker** [comment]
thanks, i just used the dataset  described  in https://github.com/rajat95/Deep-Deghosting-HDR/issues/5 to train the model from scratch, is this one the dataset you used to train the refinenet~ i want to know if the dataset and code are the same like yours~

**Comment by rajat95** [comment]
I think learning rates need a little bit playing around with to get the same result. Also quality of flow files generated can also affect the result. But the script and dataset are the same as you quoted.
